spec/factory_logger_spec.rb

```diff
@@ -1,118 +1,11 @@
-    let!(:factoryBs) { create_list(:factory_log, 3, :factoryB) }
-
-    let!(:a_sample) { factoryAs.first }
-    let!(:a_with_traits_sample) { factoryAs_with_traits.first }
-    let!(:b_sample) { factoryBs.first }
-
-    let!(:a_ranking_data) {
-                            {
-                              name: (a_sample.name + a_sample.traits),
-                              total_time: 2.0 * 10,
-                              count: 10,
-                              average_time: 2.0
-                            }
-                          }
-    let!(:a_with_traits_ranking_data) {
-                                        {
-                                          name: (a_with_traits_sample.name + a_with_traits_sample.traits),
-                                          total_time: 2.0,
-                                          count: 1,
-                                          average_time: 2.0
-                                        }
-                                      }
-    let!(:b_ranking_data) {
-                            {
-                              name: (b_sample.name + b_sample.traits),
-                              total_time: 10.0 * 3,
-                              count: 3,
-                              average_time: 10.0
-                            }
-                          }
-
-    it "ranking about time they took (total)" do
-      expect(FactoryLog.time_ranking).to eq [
-                                              b_ranking_data,
-                                              a_ranking_data,
-                                              a_with_traits_ranking_data
-                                            ]
-    end
-  end
-
-  describe ".descendants and .ancestors" do
-    let!(:top) { create :factory_log, name: "top", parent_id: nil }
-    let!(:child1) { create :factory_log, name: "child1", parent_id: top.id }
-    let!(:child2) { create :factory_log, name: "child2", parent_id: top.id }
-    let!(:child3) { create :factory_log, name: "child3", parent_id: top.id }
-    let!(:grand_son1_1) { create :factory_log, name: "grand_son1_1", parent_id: child1.id }
-    let!(:grand_son1_2) { create :factory_log, name: "grand_son1_2", parent_id: child1.id }
-    let!(:grand_son2_1) { create :factory_log, name: "grand_son2_1", parent_id: child2.id }
-
-    it "have descendants" do
-      expect(top.descendants).to eq [
-                                      [child1, [ [grand_son1_1], [grand_son1_2] ] ],
-                                      [child2, [ [grand_son2_1] ] ],
-                                      [child3]
-                                    ]
-    end
-    it "have ancestors" do
-      expect(grand_son1_1.ancestors).to eq [child1, top]
+require 'spec_helper'
+describe FactoryLogger do
+  describe "is configurable" do
+    FactoryLogger.configure do |config|
+      config.something = "something config"
+    end
+    it 'configured' do
```

Comment:
Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.